securedrop/source_app/main.py

```diff
@@ -171,6 +172,21 @@ def lookup(logged_in_source: SourceUser) -> str:
                 current_app.logger.error("Could not decode reply %s" % reply.filename)
             except FileNotFoundError:
                 current_app.logger.error("Reply file missing: %s" % reply.filename)
+            except Exception as e:
```

Comment:
I would prefer if we explicitly caught just the GpgDecryptError and RedwoodError instead of a catch-all